src/lib/viewers/media/__tests__/DashControls-test.tsx

```diff
@@ -19,13 +19,15 @@ describe('DashControls', () => {
             const onPlayPause = jest.fn();
```

Comment:
Are there any tests we need to add to this to ensure that the new components are being rendered with the correct props?